src/main/java/com/programmers/engine/validate/BracketValidator.java

```diff
@@ -0,0 +1,33 @@
+package com.programmers.engine.validate;
+
+import com.programmers.engine.stack.Bracket;
+
+import java.util.LinkedList;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class BracketValidator implements Validator{
+    AtomicInteger openBracket = new AtomicInteger();
+    AtomicInteger closeBracket = new AtomicInteger();
```

Comment:
멤버변수는 private으로 숨기는 게 어떨까요?
```suggestion
    private AtomicInteger openBracket = new AtomicInteger();
    private AtomicInteger closeBracket = new AtomicInteger();
```